doc/nrf-bm/migration/nrf5_bm_migration.rst

```diff
@@ -91,6 +87,24 @@ In |BMshort|, there are a few ready-made partitioning schemes that can be select
 Partitioning can be tweaked by making simple changes to textual **Devicetree** files which define the layout of the memory.
 These can be edited in the board files, or applied to existing boards as **overlays**.
 
+Non-volatile Memory (NVM)
+*************************
+
+The nRF51 and nRF52 devices supported by the nRF5 SDK utilized flash memory as their non-volatile memory (NVM) technology for storing program code and data.
+In contrast, the nRF54L devices employ RRAM, a different NVM technology.
+
+To support the new RRAM storage technology, the |BMshort| introduces :ref:`lib_bm_zms`, which is optimized for RRAM and replaces FDS.
+Unlike FDS, which uses a file/record ID indexing scheme based on flash pages, Zephyr Memory Storage uses a key-value indexing system that is better suited for RRAM’s access patterns and performance characteristics.
+
+It is technically feasible to reuse a file system that expects flash behavior on a device that has RRAM.
```

Comment:
This doesn't seem to help much. Perhaps a more concrete example of this? For example referencing what the DFU is using.

Reply:
I think it makes sense with the sentences below.

Reply:
I suggest to postpone improvements to this guide to next release.